### PR TITLE
use accounts rpc endpoints

### PR DESCRIPTION
**Pull request by yenda**
Accounts were previously stored within the multiaccount `setting`
serialized in transit

This moves [:multiaccount :accounts] to `:multiaccount/accounts` and uses
the `getAccounts`, `saveAccounts` and `deleteAccounts` endpoints.

## Tested

On Android (not platform dependent)
- created account
- recovered account
- transfer between account, change dapp account, account balance and transactions
status: ready

### CODE REVIEW

File: src/status_im/multiaccounts/login/core.cljs [hunk 1]
```diff
@@ -63,12 +63,30 @@
...
+   {:db (assoc db :multiaccount/accounts
+               (reduce (fn [acc {:keys [chat type wallet] :as account}]
+                         (if chat
+                           acc
+                           (let [account (cond->
+                                          (update account :address
+                                                  eip55/address->checksum)
+                                           type
+                                           (update :type keyword))]
+                             (println account)
```
Comment (rasom) [inline src/status_im/multiaccounts/login/core.cljs]: println

**Review by rasom — APPROVED** [review]

File: src/status_im/multiaccounts/login/core.cljs [hunk 2]
```diff
@@ -150,6 +168,27 @@
         update-address
         (update :accounts (partial mapv update-address)))))
 
+(re-frame/reg-fx
+ ;;TODO: this could be replaced by a single API call on status-go side
```
Comment (flexsurfer) [inline src/status_im/multiaccounts/login/core.cljs:174]: why isn't it replaced?

File: src/status_im/multiaccounts/login/core.cljs [hunk 3]
```diff
@@ -63,12 +63,30 @@
...
-  (fx/merge cofx
-            (wallet/initialize-tokens custom-tokens)
-            (wallet/update-balances nil)
-            (wallet/update-prices)
-            (transactions/initialize)))
+  [{:keys [db] :as cofx} accounts custom-tokens]
+  (fx/merge
+   cofx
+   {:db (assoc db :multiaccount/accounts
+               (reduce (fn [acc {:keys [chat type wallet] :as account}]
```
Comment (flexsurfer) [inline src/status_im/multiaccounts/login/core.cljs]: move this to function ?

**Review by flexsurfer — APPROVED** [review]

File: src/status_im/multiaccounts/login/core.cljs (hunk 2 above)
Comment (yenda) [reply src/status_im/multiaccounts/login/core.cljs:174]: For now we stick to simple RPC calls, we'll improve that next in version

**Comment by yenda** [comment]
@churik are these failed e2e tests related to the PR?

I tried transaction between accounts and transaction from dapp and didn't get these errors

**Comment by churik** [comment]
@yenda yes, they are not reproducible.

**Pull request merged**